plugins/ccgx/ccgx.quirk

```diff
@@ -9,9 +9,16 @@ Plugin = ccgx
 GType = FuCcgxHpiDevice
 
 [DeviceInstanceId=USB\VID_04B4&PID_521A&SID_1F00&APP_6D64]
+Name = ThinkPad USB-C Dock Gen2 PD Controller
 ImageKind = dual-asymmetric
 ParentGuid = USB\VID_17EF&PID_A391
 
+[DeviceInstanceId=USB\VID_04B4&PID_521A&SID_1F00&APP_6D64&MODE_FW1]
+Summary = PD-IC Backup Firmware
+
+[DeviceInstanceId=USB\VID_04B4&PID_521A&SID_1F00&APP_6D64&MODE_FW2]
+Summary = PD-IC Primary Firmware
+
```

Comment:
Could we change it such as CCGx PD Device (Primary Firmware) and CCGx PD Device (Backup Firmware)?

Reply:
Sure! I guess we could expand out PD too -- maybe `CCGx Power Delivery Device (Primary Firmware)`?

Reply:
> CCGx Power Delivery Device (Primary Firmware)

It sounds good.